### PR TITLE
progress bar done.

### PR DESCRIPTION
**Pull request by my-name-is-nheo**
@rlwong2 @oriluka @justinCalvo  just finished the progress bar...................

### CODE REVIEW

File: client/src/app.jsx [hunk 1]
```diff
@@ -13,13 +13,11 @@ class App extends React.Component {
       paused: true, // change to false if i want to autoplay on load
       trackNumber: 0,
       displayList: false,
-      displayVolume: false
+      displayVolume: false,
+      seekValue: 0
     };
```
Comment (rlwong2) [inline client/src/app.jsx:18]: Just a suggestion: maybe consider binding all your functions here in your constructor?  Easier to keep track what's been binded (bound?)

**Review by rlwong2 — APPROVED** [review]
If there's time, might be good to break out your code into components on separate pages (and pass props) instead of one long page of code.  Could be easier for people to read and separate into recognizable parts, especially whoever is running ours for SDC...

File: client/src/app.jsx (hunk 1 above)
Comment (my-name-is-nheo) [reply client/src/app.jsx:18]: make it hell for them

Comment (my-name-is-nheo) [reply client/src/app.jsx:18]: but will look into when i finish the project first

**Pull request merged**